x-pack/legacy/plugins/apm/cypress/cypress/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+module.exports = {
+  resolve: {
+    extensions: ['.ts', '.js']
+  },
+  node: { fs: 'empty', child_process: 'empty', readline: 'empty' },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        exclude: [/node_modules/],
+        use: [
+          {
+            loader: 'ts-loader'
+          }
+        ]
+      },
+      {
+        test: /\.feature$/,
```

Comment:
Files representing the features/specifications, in the form of `.feature` files.